.gitignore

```diff
@@ -17,3 +17,4 @@ android-sdk/*
 *.suo
 *.sublime-project
 *.sublime-workspace
+*.preformat.bak
```

Comment:
Please don't do that. Personally I don't like this, the reason is if you add this kind of files into gitingore, you don't have a center place to remove all of them in one batch.
For example, if there're several .bak files show in Unity and scatter in different folders, you have to go to different folders to delete all of them, it's a bit annoying to me.
For now, they all show in git (SourceTree for instance), I can delete all of them in one batch...

Wondering if the formatter tool has the option to go bold without generating the .bak files...

Reply:
updated the format script to clean up after itself 👍

Reply:
👍